### DISCUSSION TITLE
Package scikit-hts in base Environment

### DISCUSSION THREAD

**Discussion by fersebas** [PaddlePaddle/PaddleSpatial#172] [BaiduKDDCup2022] [answered]
Hello! We are using this package to forecast hiearchical time series modelling. Could you please install in the base environment?

https://scikit-hts.readthedocs.io/en/latest/installation.html

pip install scikit-hts

Thanks in advanced

**Comment by ramber1836** [comment, marked as answer]
This package has already been added to the base environment. Please check if it's OK.
